packages/taquito-michel-codec/test/contracts_014.spec.ts

```diff
@@ -31,15 +37,68 @@ describe('PtKathmandu', () => {
             expect(() => Contract.parse(src, options)).toThrow();
             return;
           }
+        });
 
-          try {
-            Contract.parse(src, options);
-          } catch (err) {
-            if (err instanceof MichelsonError) {
-              console.log(inspect(err, false, null));
-            }
-            throw err;
-          }
+        it('parse check null case', () => {
+          const options: ContractOptions = {
+            protocol: protocol,
+          };
+          const src = '';
+          expect(() => Contract.parse(src, options)).toThrow(InvalidContractError);
+          expect.objectContaining({
```

Comment:
missing something here

Reply:
i fixed all of these tests were the throw() was left out